Library/Homebrew/extend/ENV/super.rb

```diff
@@ -108,27 +98,23 @@ def cxx=(val)
   end
 
   def effective_sysroot
-    MacOS::Xcode.without_clt? ? MacOS.sdk_path.to_s : nil
   end
 
   def determine_cxx
     determine_cc.to_s.gsub("gcc", "g++").gsub("clang", "clang++")
   end
 
+  def extra_paths
```

Comment:
Any reason why this is only prefixed with `extra_` while other similar methods are prefixed with `homebrew_extra_`?